llvm/include/llvm/Support/RevngOptions.h

```diff
@@ -0,0 +1,13 @@
+#ifndef LLVM_SUPPORT_REVNGOPTIONS_H
```

Comment:
1. Add standard header comments.
2. Drop any reference to rev.ng. I prefer not to have rev.ng trickle into LLVM.
3. Move to `DebugLoc.h`?

Reply:
It's a single line addition, it's hardly going to be hard to rebase.

I'm trying to make this thing less "special".

Reply:
The commit in question is not called `Introduce a debug information preservation style`, please take a fresh look.